ext/AvizExt/viz/clustering.jl

```diff
@@ -1,8 +1,8 @@
 using Statistics
 
 """
-    scenarios(outcomes::AbstractMatrix, clusters::Vector{Int64}; opts::Dict=Dict(), fig_opts::Dict=Dict(), axis_opts::Dict=Dict())
-    scenarios!(g::Union{GridLayout,GridPosition}, outcomes::AbstractMatrix, clusters::Vector{Int64}; opts::Dict=Dict(), axis_opts::Dict=Dict())
+    scenarios(outcomes::AbstractMatrix, clusters::Vector{Int64}; opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}(), fig_opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}(), axis_opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}())

```

Comment:
Just to clarify, the idea is that the Docstring will not be identical to the function declaration in these cases then?

Reply:
As you might have guessed, I decided to just leave it

Reply:
I think it is better to leave it to what it resolves to. In the case of a new user, they will most likely miss the global type declaration and lead to more confusion. 

The global declaration allows us to easily maintain compatibility between viz functions, preventing type errors from ambiguously constructed dictionaries.